cmd/project/get.go

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2018, Google, Inc.
```

Comment:
s/2018/2019

Reply:
This should also be your company name or author name instead of Google (since we didn't write it).